tasks/add-fbc-contribution/README.md

```diff
@@ -4,13 +4,18 @@ Task to create a internalrequest to add fbc contributions to index images
 
 ## Parameters
 
-| Name           | Description                                                               | Optional | Default value        |
-|----------------|---------------------------------------------------------------------------|----------|----------------------|
-| snapshotPath   | Path to the JSON string of the mapped Snapshot spec in the data workspace | Yes      | snapshot_spec.json   |
-| dataPath       | Path to the JSON string of the merged data to use in the data workspace   | Yes      | data.json            |
-| requestTimeout | InternalRequest timeout                                                   | Yes      | 180                  |
-| binaryImage    | binaryImage value updated by update-ocp-tag task                          | No       |                      |
-| fromIndex      | fromIndex value updated by update-ocp-tag task                            | No       |                      |
+| Name           | Description                                                                                        | Optional | Default value        |
+|----------------|----------------------------------------------------------------------------------------------------|----------|----------------------|
+| snapshotPath   | Path to the JSON string of the mapped Snapshot spec in the data workspace                          | Yes      | snapshot_spec.json   |
+| dataPath       | Path to the JSON string of the merged data to use in the data workspace                            | Yes      | data.json            |
+| requestTimeout | InternalRequest timeout                                                                            | Yes      | 180                  |
+| binaryImage    | binaryImage value updated by update-ocp-tag task                                                   | No       |                      |
+| fromIndex      | fromIndex value updated by update-ocp-tag task                                                     | No       |                      |
+| pipelineRunUid | The uid of the current pipelineRun. Required for cleanup and must be passed in pipeline definition | No       |                      |
```

Comment:
I'd say you don't need to say it must be passed - that's implied by it not being optional. Also, I'd maybe say "Used for a label when creating internal requests" or something? I think that's more relevant here than the cleanup which doesn't happen in the task.

But more importantly, these are breaking changes, so the major version should go up.

Reply:
I am saying it must be passed in the pipeline definition as an explanation of why it is not in the task definition

Reply:
Updated in all tasks (major version bump and parameter description)

Reply:
OK, thanks.